Xamarin.Forms.Platform.WPF/FormsPanel.cs

```diff
@@ -82,6 +82,13 @@ protected override System.Windows.Size MeasureOverride(System.Windows.Size avail
 			if (double.IsInfinity(elementDesiredWidth) || double.IsPositiveInfinity(elementDesiredHeight))
 			{
 				Size request = Element.Measure(elementDesiredWidth, elementDesiredHeight, MeasureFlags.IncludeMargins).Request;
+
+				if (request.Width == -1)
```

Comment:
I think it would be better to check if request.Width < 0. it will cover all invalid values

Reply:
OK, did it